WalletWasabi/Tor/Http/Extensions/HttpResponseMessageExtensions.cs

```diff
@@ -97,4 +97,14 @@ public static async Task ThrowRequestExceptionFromContentAsync(this HttpResponse
 
 		throw new HttpRequestException($"{me.StatusCode.ToReasonString()}{errorMessage}");
 	}
+
+	private static ExceptionData? GetExceptionData(WabiSabiProtocolErrorCode errorCode, string data)
+	{
+		return errorCode switch
+		{
+			WabiSabiProtocolErrorCode.InputBanned or WabiSabiProtocolErrorCode.InputLongBanned => JsonConvert.DeserializeObject<InputBannedExceptionData>(data),
```

Comment:
True, fixed this by letting asp.net do the serialization. Seems to be working without a custom `JsonConverter`.

Reply:
Sorry i got new infos. It's not working. 
On backend-side we let asp.net do the serialization, but on client-side we use `Newtonsoft`. 
Until the `Error` record only contained `string`s, there were no problems. But since we changed `Error.ExceptionData` type `string` --> `ExceptionData`, the client can't deserialize the error properly.

@lontivero 
Is it time to introduce an custom `JsonConverter` for the `ExceptionData` class? 
Or can we make the client- and backend-side to be consistent in the serialization?

Reply:
As I said before you have to use the `GenericInterfaceJsonConverter` as we do for other identical cases.